### PR TITLE
Upgrade react-router to v2

### PR DESCRIPTION
**Pull request by durunvo**

### CODE REVIEW

File: src/dashboard/Dashboard.js [hunk 1]
```diff
@@ -196,64 +196,66 @@ class Dashboard extends React.Component {
       </div>
     }
 
-    return <Router history={history}>
-      <Redirect from='/' to='/apps' />
-      <Route path='/' component={App}>
-        <Route path='apps' component={AppsIndexPage} />
+    return (
```
Comment (flovilmart) [inline src/dashboard/Dashboard.js]: seems that you have an indentation problem, please revert the file

File: src/dashboard/history.js [hunk 2]
```diff
@@ -5,7 +5,8 @@
...
  */
-import createBrowserHistory from 'history/lib/createBrowserHistory';
-let history = createBrowserHistory();
 
-export default history;
\ No newline at end of file
+// Use the singleton history in react-router v2
+import { browserHistory } from 'react-router'
+
+export default browserHistory;
```
Comment (flovilmart) [inline src/dashboard/history.js]: add a trailing \N

**Comment by durunvo** [comment]
@flovilmart All fixed

**Comment by drew-gross** [comment]
Sweet, it all seems to work! Thanks for the help!

**Comment by durunvo** [comment]
@drew-gross actually I leave some syntax bug on this commit. Will submit new PR

**Comment by drew-gross** [comment]
Whoops, sorry. I assumed it was done because you removed the WIP tag, and I checked out your branch and clicked around and it did seem to work.

**Comment by durunvo** [comment]
@drew-gross The bug is in PushNew.react.js

**Comment by durunvo** [comment]
new PR #116

**Comment by drew-gross** [comment]
Cool, good thing that page isn't visible to anyone currently.

**Pull request merged**